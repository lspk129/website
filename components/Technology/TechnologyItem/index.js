import styled from 'styled-components';

const TehcnologyItem = styled.div`
  transition: all 0.5s ease-in-out;
  display: grid;
  align-items: center;
  img {
    width: 200px;
    height: 200px;
    align-self: center;
    justify-self: center;
  }
  img[alt="Next"] {
    width: 200px;
    height: 150px;
    margin: 25px 0px;
  }
  &:hover {
    transform: scale(1.2);
  }
`;

export default TehcnologyItem;
