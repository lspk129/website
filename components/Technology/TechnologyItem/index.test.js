import React from 'react';
import { shallow } from 'enzyme';
import renderer from 'react-test-renderer';

import TechnologyItem from './';

describe('TechnologyItem', () => {
  const shallowWrapper = shallow(<TechnologyItem />);

  it('renders correctly', () => {
    const wrapper = renderer.create(<TechnologyItem />);
    const tree = wrapper.toJSON();
    expect(tree).toMatchSnapshot();
  });

  it('renders div element', () => {
    expect(shallowWrapper).toBePresent();
    expect(shallowWrapper).toHaveTagName('div');
  });
});
