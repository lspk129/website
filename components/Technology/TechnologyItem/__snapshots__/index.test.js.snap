// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TechnologyItem renders correctly 1`] = `
<div
  className="sc-bdVaJa jXICNE"
/>
`;
